tests/e2e/_util.py

```
@@ -0,0 +1,33 @@
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""Shared helpers for the real-KVM end-to-end suite."""
+
+from __future__ import annotations
+
+from pathlib import Path
+
+try:
+    from smolvm_core import is_available as _core_available
+except Exception:  # pragma: no cover - native extension missing entirely
```

Ruff (0.15.15)

[warning] Do not catch blind exception: `Exception`

(BLE001)